.changeset/plenty-snakes-try.md

```diff
@@ -0,0 +1,6 @@
+---
+"@aws-amplify/ui-react-liveness": patch
+"@aws-amplify/ui-react": patch
```

Comment:
Does this affect the ui-react package, or just ui-react-liveness?

Reply:
It was just autogenerated